projects/snappy/project.yaml

```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/google/snappy"
+language: c++
+primary_contact: ""
+auto_ccs:
+  - "Adam@adalogics.com"
+sanitizers:
+  - address
+fuzzing_engines:
+  - libfuzzer
```

Comment:
Can we enable other fuzzing engines and sanitizers. undefined, ; afl and honggfuzz ?

Reply:
Yes, that should not be a problem. The reason for the current set up is that changes are needed upstream in the build system to support that. Once the maintainers confirm interest in oss-fuzz integration, I will sort it out.